nixos/modules/services/misc/gerrit.nix

```diff
@@ -0,0 +1,158 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.gerrit;
+
+  cmd = "${pkgs.jre_headless}/bin/java -jar ${cfg.package}/webapps/gerrit-${cfg.package.version}.war";
```

Comment:
I'm into it, I've definitely found it annoying to have to dig the command out of `systemctl cat` to check stuff while debugging.

Reply:
pushed to your PR as 17d73b82b0823efa641c8dd9fc04f2a0a4599fea

I also added a jvmOpts module option because it because quickly necessary to tune the JVM for larger gerrit instances.